src/ekat/ekat_pack_kokkos.hpp

```diff
@@ -229,11 +299,31 @@ repack_impl (const Kokkos::View<OldPackT**, ViewProps...>& vp) {
 // General access point for repack (calls one of the three above)
 template <int N, typename OldPackT, typename... ViewProps>
 KOKKOS_FORCEINLINE_FUNCTION
-Unmanaged<Kokkos::View<typename RepackType<N,OldPackT>::type**,ViewProps...>>
+typename std::enable_if<IsPack<OldPackT>::value,
+    Unmanaged<Kokkos::View<typename RepackType<N,OldPackT>::type**,ViewProps...>>
+>::type
 repack (const Kokkos::View<OldPackT**, ViewProps...>& vp) {
   return repack_impl<typename RepackType<N,OldPackT>::type>(vp);
 }
 
+// 2d pack from a scalar view
+template <int N, typename ScalarT, typename... ViewProps>
+KOKKOS_FORCEINLINE_FUNCTION
+typename std::enable_if<!IsPack<ScalarT>::value,
```

Comment:
This is why I needed `IsPack`. For the "positive" case I could have used `OnlyPackReturn<Unmanaged<...>>`, but for the negative case I had no tool to use. I could not do
```
enable_if<not ScalarT::packtag>
```
cause that fails if ScalarT is not a pack (which is the case I need).